_content/monographs/place-of-animals-in-buddhism_story-francis.md

```diff
@@ -0,0 +1,17 @@
+title: "The Place of Animals in Buddhism"
+authors:
+  - "Francis Story"
+external_url: "https://www.bps.lk/olib/bl/bl023_Story_The-Place-of-Animals-in-Buddhism.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/11hFKwwisU4i7FMdSITab_trwgfr2dYZK/view?usp=sharing"
+course: animals
+tags:
+  - karma
+  - rebirth
+month: 1964
+publisher: bps
```

Comment:
Yes, monographs since there's a publisher is fine. For Bodhi Leaves, we don't yet have a proper slug for it, so just note it as:

```yaml
series: "Bodhi Leaves"
number: 23
```
Once we have a few more Bodhi Leaves added, I'll make a page for them. :smile: